catalog-rest-service/src/main/java/org/openmetadata/catalog/resources/feeds/FeedResource.java

```diff
@@ -187,6 +203,22 @@ public Response addPost(@Context UriInfo uriInfo, @PathParam("id") String id, @V
     return Response.created(thread.getHref()).entity(thread).build();
   }
 
+  @GET
+  @Path("/{id}/posts")
+  @Operation(
+      summary = "Get all the posts of a thread",
+      tags = "feeds",
+      description = "Get all the posts of an existing thread.",
+      responses = {
+        @ApiResponse(
+            responseCode = "200",
+            description = "The posts of the given thread.",
+            content = @Content(mediaType = "application/json", schema = @Schema(implementation = PostList.class))),
+      })
+  public PostList getPosts(@Context UriInfo uriInfo, @PathParam("id") String id) throws IOException {
```

Comment:
I'm trying to interpret the terminology here (feed, thread and post)
A thread has many posts. 

Is feed and thread the same and used interchangeably ?

Reply:
I understand that it is a bit confusing. A feed contains a bunch of threads and a thread contains a bunch of posts. Feed is just virtual and doesn't get an id for itself. And that's why the APIs take thread id for `/api/v1/feed/{id}`.

Reply:
Thanks for clarifying. Was asking to confirm my own understanding.